parsons/google/google_sheets.py

```diff
@@ -108,6 +108,50 @@ def read_sheet_with_title(self, spreadsheet_id, title):
         index = self.get_sheet_index_with_title(spreadsheet_id, title)
         return self.read_sheet(spreadsheet_id, index)
 
+    def share_spreadsheet(self, spreadsheet_id, sharee, share_type='user', role='reader',
+                          notify=True, notify_message=None, with_link=False):
+        """
+        Share a spreadsheet with a user.
```

Comment:
for clarity, i would edit this to say "Share a spreadsheet with a user and/or the public." I had wished gspread was more clear in their description of "share()" to explicitly say that you can change the overall permissions of the sheet. Open to your thoughts though!

Reply:
Great suggestion!